src/ui.py

```diff
@@ -6,7 +6,7 @@
 from graphs import file_io, graphs
 
 
-def toggle_darkmode(_shortcut, _theme, _widget, self):
+def on_style_change(_shortcut, _theme, _widget, self):
```

Comment:
Makes sense with the renaming in GNOME 44 from Dark Mode to "Dark Style"

![image](https://user-images.githubusercontent.com/68477016/229055919-64dccfc6-7f2f-484d-a639-975986c2d2f4.png)